### PR TITLE
fix #31, elements fade to opacity 0

### LINKED ISSUE #31: map content fade to 0%
client says images and videos should fade out to 0% when hovering text. atm videos aren't fading at all i think

### PR DESCRIPTION
**Pull request by mixcas**

### CODE REVIEW

**Comment by mikeyrayvon** [comment]
i fuck with this branch name 💰  @mixcas

File: src/styl/responsive/mobile.styl [hunk 1]
```diff
@@ -49,6 +49,7 @@ $main-content-offset = $margin-small + $margin-mid
 
 #main-content
   width: 360px
+  transition: opacity $fade-duration
```
Comment (mikeyrayvon) [inline src/styl/responsive/mobile.styl:52]: y tho? cause line `48` already does this work, my dude
🍂

Comment (varffs) [reply src/styl/responsive/mobile.styl:52]: This doesn't seem to be true. Testing my end removing this line means only the images transition but the text doesnt

**Review by varffs — APPROVED** [review]
tbh I don't really get it tho cus shouldn't the transition be declared on the element that transitions not the parent? I didn't think those things were inherited?

File: src/styl/responsive/mobile.styl (hunk 1 above)
Comment (mixcas) [reply src/styl/responsive/mobile.styl:52]: yeah, thats why i had to copy it here

**Comment by varffs** [comment]
closes #31

**Pull request merged**